### DISCUSSION TITLE
Alterar Relatórios

### DISCUSSION THREAD

**Discussion by LucasDapper** [RamonSilva20/mapos#2165] [Q&A]
Bom dia!

O sistema é maravilhoso, estou curtindo muito, porem gostaria de saber se tem possibilidade de alterar alguns relatórios, deixa-los esteticamente mais bonitos e organizados, teria essa possibilidade ?

**Comment by Eaglemcz** [comment]
Realmente, o sistema está excelente, mas quero também implementar relatórios gráficos, a pedido de alguns clientes. seria interessante essa melhoria.
